tools/wptrunner/wptrunner/testrunner.py

```diff
@@ -402,10 +402,26 @@ def run(self):
         except Exception:
             message = "Uncaught exception in TestRunnerManager.run:\n"
             message += traceback.format_exc()
-            self.logger.error(message)
+            self.logger.critical(message)
             raise
         finally:
             self.logger.debug("TestRunnerManager main loop terminating, starting cleanup")
+
+            skipped_tests = []
+            while True:
+                _, _, test, _, _ = self.get_next_test()
+                if test is None:
+                    break
+                skipped_tests.append(test)
+
+            if skipped_tests:
+                self.logger.critical(
+                    f"Tests left in the queue: {skipped_tests[0].id!r} "
+                    "and {len(skipped_tests)} others"
```

Comment:
I believe you need an `f` before this string to make interpolation work. Presumably it's also `len(skipped_tests) - 1` others although you could just write something like `f"{len(skipped_tests)} tests left in the queue (first: {skipped_tests[0].id!r}."`